src/self.h

```diff
@@ -4,6 +4,8 @@
 
 // some info taken from the wiki, see http://vitadevwiki.com/index.php?title=SELF_File_Format
 
+const char digest_constant[] =  {0x62, 0x7C, 0xB1, 0x80, 0x8A, 0xB9, 0x38, 0xE3, 0x2C, 0x8C, 0x09, 0x17, 0x08, 0x72, 0x6A, 0x57, 0x9E, 0x25, 0x86, 0xE4};
```

Comment:
`uint8_t`

Reply:
And add to size. like `digest_constant[0x14]`.